contracts/teller-network/ArbitratorLicense.sol

```diff
@@ -0,0 +1,93 @@
+
+pragma solidity ^0.5.8;
+
+import "./License.sol";
+
+/**
+* @title ArbitratorLicense
+* @dev Contract for management of an arbitrator license
+*/
+contract ArbitratorLicense is License{
+
+    enum RequestStatus {AWAIT,ACCEPTED,REJECTED}
+
+    struct Requests{
+        address seller;
+        address arbitrator;
+        RequestStatus status;
+    }
+
+	struct ArbitratorLicenseDetails {
+        uint id; 
+        bool acceptAny; // accept any seller
+        address[] accepted; // addresses of accepted sellers
+    	bool isActive; // is license active?
+    }
+
+    mapping(address => ArbitratorLicenseDetails) arbitratorlicenseDetails;
+
+    Requests[] public requests;
+    mapping(uint => uint[]) public requestsById;
+
+    event ArbitratorRequested(uint id, address seller, address arbitrator);
+    event ArbitratorLicensed(uint id, bool acceptAny);
+    event RequestAccepted(address arbitrator, address seller);
+
+    /**
+     * @notice Buy an arbitrator license
+     * @param _acceptAny indicates does arbitrator allow to accept any seller/choose sellers 
+     */
+    function buyLicense(bool _acceptAny) public {
+    	uint _id = buy();
+    	address[] memory addresses;
+
+        arbitratorlicenseDetails[msg.sender] = ArbitratorLicenseDetails({
+            id: _id,
+            acceptAny: _acceptAny,
+            accepted: addresses,    
+ 			isActive:  true			
+        });
+
+        emit ArbitratorLicensed(_id, _acceptAny);
+    }
+
+
+    /**
+     * @notice Allows arbitrator to accept a seller
+     * @param _arbitrator address of a licensed arbitrator
+     */
+    function requestArbitrator(address _arbitrator) public {
+       require(arbitratorlicenseDetails[_arbitrator].isActive, "Arbitrator should have a valid license");
+       require(!arbitratorlicenseDetails[_arbitrator].acceptAny, "Arbitrator already acceps all cases");
+       require(isLicenseOwner(msg.sender), "Must be a valid seller to fund escrow transactions");
```

Comment:
This will check if the sender has an arbitrator License. You need to have a reference to the seller licenses and then do a require `licenses.isLicenseOwner(msg.sender)` with a proper error message

Reply:
actually it should check if the message sender has a seller license, sorry for the error message, will fix it